File: msg-entity/src/main/java/dev/number6/entity/ChannelMessagesEntityRequestHandler.java
package dev.number6.entity;

import dev.number6.entity.dagger.DaggerChannelMessagesEntityComprehensionComponent;
import dev.number6.message.ChannelMessagesNotificationRequestHandler;

public class ChannelMessagesEntityRequestHandler extends ChannelMessagesNotificationRequestHandler {

    public ChannelMessagesEntityRequestHandler() {
        super(DaggerChannelMessagesEntityComprehensionComponent.create().getChannelMessagesHandler());
    }
}

File: msg-entity/src/main/kotlin/dev/number6/entity/ChannelMessagesEntityRequestHandler.kt
package dev.number6.entity

import dev.number6.entity.dagger.DaggerChannelMessagesEntityComprehensionComponent
import dev.number6.message.ChannelMessagesNotificationRequestHandler

class ChannelMessagesEntityRequestHandler :
        ChannelMessagesNotificationRequestHandler(DaggerChannelMessagesEntityComprehensionComponent.create().getChannelMessagesHandler())